index.html

```diff
@@ -0,0 +1,1591 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="main.css" />
+    <title>TNW</title>
+    <script
+      src="https://kit.fontawesome.com/63c1245c39.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!-- Navigator -->
+    <nav class="nav nav-m-b">
```

Comment:
In the mobile view, the hamburger icon is missing from the navbar.

| Source Page	| Clone Page |
|:---:|:---:|
|![Annotation 2020-07-10 030306](https://user-images.githubusercontent.com/57726348/87093032-244b9600-c25a-11ea-9728-5c12924bd473.jpg)|![image](https://user-images.githubusercontent.com/57726348/87092987-10079900-c25a-11ea-8a07-5ba01bba78ca.png)|